.github/workflows/mirror_community_pipeline.yml

```diff
@@ -0,0 +1,54 @@
+name: Fast tests for PRs
+
+on:
+  # Push changes on the main branch
+  push:
+    branches:
+      - main
+    paths:
+      - 'examples/community/**.py'
+
+  # And on tag creation
+  create:
+    tags:
+      - '*'
+jobs:
+  mirror_community_pipeline:
+    runs-on: ubuntu-latest
+    steps:
+      # Setup + install dependencies
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install uv
+          uv pip install --upgrade huggingface_hub
+
+      # Determine to which folder to push changes (based on branch or tag)
+      - name: Infer path in repo (main or tag)
+        run: |
+          if [ "${GITHUB_EVENT_NAME}" == "push" ]; then
+            if [ "${GITHUB_REF}" == "refs/heads/main" ]; then
+              echo "PATH_IN_REPO=main" >> $GITHUB_ENV
+            else
+              echo "PATH_IN_REPO=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+            fi
+          elif [ "${GITHUB_EVENT_NAME}" == "push" ] && [ "${GITHUB_REF}"!= "refs/heads/main" ]; then
+            echo "PATH_IN_REPO=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          fi
```

Comment:
Would be nice provide an example for `PATH_IN_REPO` in the comment for the different conditions we're adding here.

Reply:
Good idea. Added an example in https://github.com/huggingface/diffusers/pull/8417/commits/cb0bf321aceecd6e1c9d5328dfca4b30998494d0